File: Lab2/src/main/scala/Coffee/SyrupCappuccino.scala
package Coffee

class SyrupCappuccino(intensity: Intensity, mlOfWater: Int, mlOfMilk: Int, val syrup: SyrupType)
  extends Cappuccino(intensity, mlOfWater, mlOfMilk):

  private val coffeeName: String = "Syrup Cappuccino"

  def getSyrup: SyrupType = syrup

  override def printCoffeeDetails(): Unit =
    println(s"$coffeeName water: ${getMlOfWater} ml, milk: ${mlOfMilk} ml, syrup: $syrup")

  override def makeRecipe(): Unit =
    super.makeRecipe()
    println(s"Adding $syrup syrup")

File: lab2/src/main/kotlin/labo2/Task1/SyrupCappuccino.kt
package org.example.labo2.Task1

class SyrupCappuccino(
    private var intensityCoffe: Intensity,
    private var mltrOfMilk: Int,
    private var syrup: SyrupType,
    private val coffee: String = "SyrupCappuccino"
) : Cappuccino(intensityCoffe, mltrOfMilk, coffee) {
    //pass
}